### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by erp78** [YatingRu/ORIE5741_project#13]
This project is about predicting child undernutrition in Ethiopia. It is framed as a classification problem on whether or not a child is stunted. Household survey data as well as satellite data is used. The types of models fit are logistic regression, random forest, and ControlBurn.

You did a great job, both in the report and the video, in providing background information on the child undernutrition issue in Ethiopia and on the datasets that you used. This framed a clear objective for your project. Next, I like your data visualizations that plot the stunting level versus some features of interest, as well as the accompanying analysis on them. They do appear to reveal some trends, which motivates the project. Lastly, I think your Models section was strong and well-organized. You laid out the types of models you experimented with, explained your decision making process, and reported what regularization/hyperparameter tuning you included.

I understand that you were interested in comparing the performances of satellite-derived data and survey-based data, but I think you could've also considered combining the features from both datasets to see how accurate of a prediction could be made with all the available information. This might help with increasing the prediction accuracy, which you identify as being a need before you can confidently predict in practice. Another area of improvement could be to discuss more about your train-test split. It would've been nice to report your accuracy scores on the train set in addition to the test results to be able to evaluate overfitting/generalization of the models. Finally, I think you could've gone into more detail about your imputation decisions.

Overall this report was very detailed and organized, and your modeling approach seems trustworthy.
